sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/package.scala

```diff
@@ -359,7 +359,18 @@ package object expressions  {
 
         case ambiguousReferences =>
           // More than one match.
-          val referenceNames = ambiguousReferences.map(_.qualifiedName).mkString(", ")
+          var referenceNames = ""
+          if (ambiguousReferences.map(_.qualifiedName).toSet.size == 1) {
```

Comment:
Can you add a test?

Reply:
Also, `SQLQueryTestSuite` is failing, so you many want to check.

Reply:
I have updated relevant out files under sql/core/src/test/resources/sql-tests/results/postgreSQL